docs/coordinates/velocities.rst

```diff
@@ -2,14 +2,14 @@
 
 .. _astropy-coordinates-velocities:
 
-Working with velocities in Astropy coordinates
+Working with Velocities in Astropy Coordinates
 **********************************************
 
-Using velocities with ``SkyCoord``
+Using Velocities with ``SkyCoord``
 ==================================
 
-The easiest way of getting a coordinate object with velocities is to use the
-|skycoord| interface.  For example, a |skycoord| to represent a star with a
+One recommended way of getting a coordinate object with velocities is to use the
```

Comment:
As above in the generic `SkyCoord` discussion, the connotation intend here is more like "really the way you ought to start with if you aren't sure you need something else" rather than "one among many possible ways".  So what about:

```suggestion
The best way to start getting a coordinate object with velocities is to use the
```
In parallel with my suggestion above?